sdk/servicebus/service-bus/samples/README.md

```diff
@@ -17,6 +17,16 @@ npm install @azure/service-bus
 
 Before running any of the samples, update it with the connection string and the queue/topic/subscription names you have noted down above.
 
+## Initializing the samples folder
+
+Once you clone this repository, make sure you're in this samples folder. If you're already in this folder you might ignore this step, otherwise do as follows:
```

Comment:
Cloning the repository should not be the suggested way of running the samples. We have a huge repo and it will only grow bigger.

The default suggestion should always be to copy just the sample the user is interested and then user running `npm install @azure/service-bus`  in their folder.